python/paddle/fluid/layers/nn.py

```diff
@@ -1901,6 +1902,72 @@ def sequence_last_step(input):
     return sequence_pool(input=input, pool_type="last")
 
 
+def sequence_slice(input, offset, length, name=None):
+    """
+    **Sequence Slice Layer**
+
+    The layer crops a subsequence from given sequence with given start 
+    offset and subsequence length.
+
+    It only supports sequence data (LoDTensor with lod_level equal to 1).
+
+    .. code-block:: text
+    
+	- Case:
+            Given the input Variable **input**,
+	        input.data = [[a1, a2], [b1, b2], [c1, c2], [d1, d2], [e1, e2]],
+	        input.lod = [[0, 3, 5]], input.dims = (5, 2)
```

Comment:
many public APIs, like create_lod_tensor uses length-based lod. Should the API docs be updated to length-based?

Reply:
Agree and I made the change. Seems that now both offset-based and length-based LoD exist in the Python APIs, we'd better make them unified.